frontend/src/components/UserInfoCard.module.scss

```diff
@@ -35,3 +35,9 @@
   @extend .layout-row;
   min-width: 400px;
 }
+
+.passwordButton {
+  @extend .button-small-filled;
+  justify-content: center;
+  display: flex;
+}
```

Comment:
Please, configure your editor to add newline at the end of the file. That has been the practice on the project, and it also reduces unnecessary noise in git diffs.

Also the indentation is inconsistent in the other css file.

Just formatting the files with prettier should work

Reply:
Now I added automatic linebreaks to vscode, and fixed indentation to 2 spaces.